.github/workflows/build-dev.yml

```diff
@@ -0,0 +1,50 @@
+name: Development build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    env:
+      NODE_OPTIONS: --max_old_space_size=4096
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        node_version: [10, 12, 14]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use node.js
+      uses: actions/setup-node@v1
+      with:
+        noe-version: ${{ node_version }}
+    - name: Install dependencies
+      run: npm install
+    - name: Prepare environment
+      run: |
+        npm install -g typescript
+        npm install -g gulp
+        npm install -g vsce
+    - name: Build
+      run: npm run compile
+    - name: Lint extension source
+      run: gulp tslint
+    # TODO: smoke test
+    - name: Package extension
+      if: github.event_name == 'push' && matrix.os == 'ubuntu-latest' && matrix.node_version = 14
+      run: |
+        vsce package
+        mkdir vsix
+        mv *.vsix vsix
+    - name: Archive extension
```

Comment:
I was thinking for testing or 'experimental build' purposes.  Like if a user reports a bug, and we have a candidate fix but aren't sure if it works, we can direct the user to install the VSIX from the build, rather than requiring them to build from source.